pkg/compiler/codegen.go

```diff
@@ -1075,6 +1075,19 @@ func (c *codegen) convertBuiltin(expr *ast.CallExpr) {
 		} else {
 			c.prog.Err = errors.New("panic should have string or nil argument")
 		}
+	case "delete", "Remove":
```

Comment:
Won't this extend `delete` to work with slices or it's covered by parser already?

Reply:
Yes, error is raised by parser.